packer/scripts/centos/slurm.sh

```diff
@@ -86,12 +86,9 @@ fi
 # Install Enroot
 #
 
-ENROOT_VERSION_FULL=${1:-3.4.0-2}
-ENROOT_VERSION=${ENROOT_VERSION_FULL%-*}
-
 arch=$(uname -m)
-rpm -q enroot || yum install -y https://github.com/NVIDIA/enroot/releases/download/v${ENROOT_VERSION}/enroot-${ENROOT_VERSION_FULL}.el7.${arch}.rpm
-rpm -q enroot+caps || yum install -y https://github.com/NVIDIA/enroot/releases/download/v${ENROOT_VERSION}/enroot+caps-${ENROOT_VERSION_FULL}.el7.${arch}.rpm
+yum install -y https://github.com/NVIDIA/enroot/releases/download/v3.4.1/enroot-3.4.1-1.el8.${arch}.rpm
```

Comment:
Is it expected to have el8 on centos 7

Reply:
yes, there is no el7 in this release. el8 works on both